ray-operator/controllers/ray/common/pod_test.go

```diff
@@ -523,6 +523,22 @@ func TestBuildPodWithAutoscalerOptions(t *testing.T) {
 	customEnv := []v1.EnvVar{{Name: "fooEnv", Value: "fooValue"}}
 	customEnvFrom := []v1.EnvFromSource{{Prefix: "Pre"}}
 
+	// Define a custom security profile.
+	allowPrivilegeEscalation := false
+	capabilities := v1.Capabilities{
+		Drop: []v1.Capability{"All"},
```

Comment:
I am not sure whether "All" is equivalent with "ALL". In my experience, `drop: ["ALL"]` works, but `drop: ["all"]` does not work.

Reply:
Fixed.